packages/instrumentation-express/package.json

```diff
@@ -31,21 +31,21 @@
         "access": "public"
     },
     "dependencies": {
-        "@opentelemetry/api": "^1.0.0-rc.0",
-        "@opentelemetry/instrumentation": "^0.19.0",
-        "@opentelemetry/semantic-conventions": "^0.19.0"
+        "@opentelemetry/api": "^0.20.0",
+        "@opentelemetry/instrumentation": "^0.20.0",
+        "@opentelemetry/semantic-conventions": "^0.20.0"
```

Comment:
you need to add a dependency on `@opentelemetry/core` for the `getRPCMetadata` import